ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/CheckDetailsPage.tsx

```diff
@@ -90,15 +101,24 @@ function CheckDetails() {
             <Divider component="div" />
             <PageSection variant="light">
                 <DetailsPageHeader
-                    isLoading={isLoading}
+                    isLoading={isLoadingCheckStats}
                     name={checkName}
                     labels={checkStatsLabels}
-                    summary={data?.rationale}
+                    summary={checkStats?.rationale}
                     nameScreenReaderText="Loading profile check details"
                     metadataScreenReaderText="Loading profile check details"
+                    error={checkStatsError}
+                    errorAlertTitle="Unable to fetch profile check stats"
                 />
             </PageSection>
             <Divider component="div" />
+            <PageSection>
+                <CheckDetailsTable
```

Comment:
I guess I still have the same concerns from a previous PR here. Check out the comment in that PR https://github.com/stackrox/stackrox/pull/11082/files#r1605256355. What are your thoughts on that? To me, it feels like an unnecessary abstraction. I like the idea of keeping things in line but abstracting smaller sections of the table. So the distinction could be like this:

```
<Page>
    <PageHeader />
    <Table />
</Page>
```

```
<Page>
    <PageHeader />
    <Table>
        <TableHead>
            <Tr>
                <Th>...</Th>
                <Th>...</Th>
                <Th>...</Th>
            </Tr>
        </TableHead>
        <TableBody>
           <Tr>
                <Td>
                    <ClusterNameLink ... />
                </Td>
                <Td>
                    <LastScannedDate ... />
                </Td>
                <Td>
                     <ComplianceStatus ... />
                </Td>
            </Tr>
        </TableBody>
    </Table>
</Page>
```

If we ever need to make changes to specific areas, it'll be easy to locate and change. If we have multiple tables that look very similar and have slight differences, we can account for that by constructing the table again but pick and choose the components for the similar columns.

Just wanted to offer a different perspective 👀